recipes/ux.rb

```diff
@@ -8,12 +8,19 @@
 #
 
 log "Create jellyfish user"
-
 user "jellyfish" do
   comment "jellyfish user"
   shell "/bin/bash"
 end
 
+log "Create jellyfish sudoers file"
+file "/etc/sudoers.d/jellyfish" do
```

Comment:
I was unable to get the jellyfish user to startup node correctly without using the sudo command. 
The forever app wouldn't startup the second node process. Just the first forever/bin/monitor app.js process would
start and the start page would not render. 

 3544 ?        Ssl    0:00 /usr/bin/node /usr/lib/node_modules/forever/bin/monitor app.js
 3546 ?        Sl     0:00 /usr/bin/node /home/jellyfish/ux/app.js